cli/index.js

```diff
@@ -0,0 +1,204 @@
+// This is a CLI tool, using console is OK
+/* eslint no-console: 0 */
+const {spawn, exec} = require('child_process');
+const {isAbsolute, resolve} = require('path');
+const {existsSync} = require('fs');
+const pify = require('pify');
+const args = require('args');
+const chalk = require('chalk');
+const opn = require('opn');
+// const columnify = require('columnify');
+// const got = require('got');
+// const ora = require('ora');
+const api = require('./api');
+
+let commandPromise;
+
+const checkConfig = () => {
+  if (api.exists()) {
+    return true;
+  }
+  let msg = chalk.red(`Error! Config file not found: ${api.configPath}\n`);
+  msg += 'Please launch Hyper and retry.';
+  console.error(msg);
+  process.exit(1);
+};
+
+args.command(['i', 'install'], 'Install a plugin', (name, args_) => {
+  checkConfig();
+  const plugin = args_[0];
+  commandPromise = api
+    .install(plugin)
+    .then(() => console.log(chalk.green(`${plugin} installed successfully!`)))
+    .catch(err => console.error(chalk.red(err)));
+});
+
+args.command(['u', 'uninstall', 'rm', 'remove'], 'Uninstall a plugin', (name, args_) => {
+  checkConfig();
+  const plugin = args_[0];
+  commandPromise = api
+    .uninstall(plugin)
+    .then(() => console.log(chalk.green(`${plugin} uninstalled successfully!`)))
+    .catch(err => console.log(chalk.red(err)));
+});
+
+args.command(['ls', 'list'], 'List installed plugins', () => {
+  checkConfig();
+  let plugins = api.list();
+
+  if (plugins) {
+    console.log(plugins);
+  } else {
+    console.log(chalk.red(`No plugins installed yet.`));
+  }
+  process.exit(1);
+});
+/*
+const lsRemote = () => {
```

Comment:
Remove?

Reply:
It will be reactivated when NPM API request (URL and criteria) will be chosen.

Reply:
fixed